tests/acceptance/blocking_adapter_test.py

```diff
@@ -271,6 +270,16 @@ def test(self):
         self.assertFalse(ch._impl._consumers)
 
 
+class TestUsingInvalidQueueArgument(BlockingTestCaseBase):
+    def test(self):
+        """BlockingConnection raises expected connection when invalid queue parameter is used
+        """
+        connection = self._connect()
+        ch = connection.channel()
+        with self.assertRaises(AssertionError):
```

Comment:
Surprising that the underlying implementation raises AssertionError instead of TypeError/ValueError. But no need to fix that.